includes/steem.php

```diff
@@ -351,13 +351,8 @@ public function _comment($parentAuthor, $parentPermlink, $author, $permlink, $ti
    */
   public function createPost($author, $title, $body, $tags = null, $app = null, $jsonMetadata = null, $permlink = null)
   {
-    if (empty($tags) && function_exists('get_option')) {
-      $tags = get_option("steem_dapp_default_tags");
-    }
-    if (!empty($tags) && is_string($tags)) {
-      $tags = strtolower($tags);
-      $tags = wp_parse_list($tags);
-    }
+    $tags = $this->collectTags($tags);
```

Comment:
We'd better move `collectTags()` function to `post.php`.

Because we have another user-level setting ** user_steem_default_tags** (
 https://github.com/steem-aksai/steem4wp/blob/96279fcc8ecb3bc5659469d95e58bcb5b9d43a15/includes/hooks.php#L133), which is only used by WordPress authors (but not WeChat Mini-Program users. The logic of processing tags a bit different for these two scenarios (WordPress authors v.s. WeChat Mini-Program users). 

For WordPress authors, the tags should be a union set of (post-level tags + user-level default tags), and may ignore the admin-level default tags to provide authors more freedom.